lib/sequel/adapters/vertica.rb

```diff
@@ -32,7 +32,6 @@ def connect(server)
           :user => opts[:user],
           :password => opts[:password],
           :port => opts[:port],
-          :schema => opts[:schema],
```

Comment:
The latest docs at http://www.rubydoc.info/gems/vertica/1.0.0 do not list `:schema` as a valid connection option, and it will in fact error out if you try to provide it. An option is to detect if it's passed in, and add a `SET search_path TO #{schema}` statement after connect, but it's not natively supported anymore.